alembic/versions/1ea1e8070ac8_add_assignee.py

```diff
@@ -0,0 +1,43 @@
+"""add assignee
+
+Revision ID: 1ea1e8070ac8
+Revises: adb90a264e3
+Create Date: 2016-04-09 11:53:52.668646
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '1ea1e8070ac8'
+down_revision = 'adb90a264e3'
+
+from alembic import op
+import sqlalchemy as sa
+from coaster.sqlalchemy import JsonDict
+import sqlalchemy_utils
+
+
+def upgrade():
+    op.create_table('assignee',
+        sa.Column('created_at', sa.DateTime(), nullable=False),
+        sa.Column('updated_at', sa.DateTime(), nullable=False),
+        sa.Column('fullname', sa.Unicode(length=80), nullable=False),
+        sa.Column('email', sa.Unicode(length=254), nullable=False),
+        sa.Column('phone', sa.Unicode(length=80), nullable=True),
+        sa.Column('details', JsonDict(), server_default='{}', nullable=False),
+        sa.Column('previous_id', sa.Integer(), nullable=True),
+        sa.Column('user_id', sa.Integer(), nullable=True),
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.ForeignKeyConstraint(['previous_id'], ['assignee.id'], ),
+        sa.PrimaryKeyConstraint('id'),
```

Comment:
Foreign key for user_id is missing. You need a user table first, so merge Sindhu's PR if necessary.